File: EASY/PRIME-TESTING/pt.go
package main

import (
	"fmt"
	"math"
)

func isPrime(n int) bool {
	if n <= 1 {
		return false
	}
	for i := 2; i <= int(math.Sqrt(float64(n))); i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	var n int
	fmt.Print("Enter a number: ")
	fmt.Scan(&n)
	if isPrime(n) {
		fmt.Println("YES")
	} else {
		fmt.Println("NO")
	}
}

File: EASY/PRIME-TESTING/pt.py
class Solution:
    def is_prime(self,n):
        if n<=1:
            return False
        for i in range(2, int(n**0.5) + 1):
            if n%i == 0:
                return False
        return True

class main:
    def __init__(self):
        s = Solution()
        n = int(input("Enter a number: "))
        if s.is_prime(n):
            print("Yes")
        else:
            print("No")

if __name__ == "__main__":
    main()